test/SIL/Serialization/dynamically_replaceable.sil

```diff
@@ -2,7 +2,7 @@
 // RUN: %target-swift-frontend %s -emit-module -o %t/tmp.swiftmodule
 // RUN: %target-sil-opt %t/tmp.swiftmodule -emit-sorted-sil | %FileCheck %s
 
-sil [serialized] [dynamically_replacable] @test_dynamically_replaceable : $@convention(thin) () -> () {
+sil [serialized] [dynamically_replaceable] @test_dynamically_replaceable : $@convention(thin) () -> () {
```

Comment:
https://github.com/swiftlang/swift/pull/75029#discussion_r1675987651

Reply:
reverted (two occurrences)

Reply:
You missed [these](https://github.com/swiftlang/swift/pull/75033/files#diff-9fce9cb6a18275d674638df67df33562c4d4363773267b5812fc8ba092721f38).

Reply:
Fixed now